helmfile/charts/gatekeeper/podsecuritypolicies/templates/default/mutations/run-as-non-root.yaml

```diff
@@ -39,7 +50,43 @@ spec:
 apiVersion: mutations.gatekeeper.sh/v1
 kind: Assign
 metadata:
-  name: mutation-init-run-as-non-root
+  name: run-as-non-root-ephemeral
```

Comment:
@robinAwallace Should I add ephemeral for all of these?

And I was thinking about changing the name since we already know all are mutations based on their kind, and if we suffix rather then prefix then all groups should be kept alphabetically.

Reply:
1. Please do :slightly_smiling_face: 
2. Yeah I like that idea actually. :+1: